plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeReadTimestamps.java

```diff
@@ -106,6 +107,7 @@ public void registerTables()
     }
 
     @Test
+    @Disabled("This test assumes specific tzdb version and it will fail on mismatch between Spark and Trino tzdb versions")
```

Comment:
isn't this a user-facing issue then?

Reply:
What if we update the JDK used to run Spark to the latest patch release for that version?